posts/posts.11tydata.js

```diff
@@ -0,0 +1,19 @@
+const {titleCase} = require("title-case");
+const path = require("path");
+
+module.exports = {
+    layout: "default",
+    tags: ["posts"],
+    eleventyComputed: {
+        
+        title: data => {
+            if(data.title){
+                return data.title;
+            }
+            let str = titleCase(data.page.fileSlug);
+            str = str.replace(/-/g, ' ');
```

Comment:
Should these be reversed? Before you add in spaces, the `str` variable is a single word. Only the first letter is capitalized.

Reply:
Yes you're right - sorry! Will fix

Reply:
Netlify is complaining. Try this:
```js
let str = data.page.fileSlug.replace(/-/g, ' ');
str =  titleCase(str);
```